new-relic/__tests__/new-relic.spec.js

```diff
@@ -103,7 +105,9 @@ describe('New Relic', () => {
       crashCollectorAddress: "gov-mobile-crash.newrelic.com",
       fedRampEnabled: true,
       nativeCrashReportingEnabled:false,
-      offlineStorageEnabled:false
```

Comment:
As far as.I know, fedRamp, offlineStorage, newEventSystem, and backgroundReporting should be disabled by default.

Reply:
this is for testing.